Задача 2. Проведите тест гипотезы. Утверждается, что шарики для подшипников, изготовленные
автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из
n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [3]:
M = 17
a = 0.05
Z_t = 1.65
n = 100
M_x = 17.5
D = 4
res = (M_x - M)/(2/10)
res

2.5

Вывод

Нулевая гипотеза, что средний диаметр шариков равен 17 мм отвергается

Альтернативня гипотеза, что ср.диаметр больше 17 мм (2.5 > 1.65) не отвергается